sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -51,12 +51,14 @@ With above configuration, `azure` client can be authenticated by following code:
 TokenCredential credential = new DefaultAzureCredentialBuilder()
     .build();
 // create client using DefaultAzureCredential
-LoadTestingClient client = new LoadTestingClientBuilder()
+LoadTestingClientBuilder builder = new LoadTestingClientBuilder()
     .credential(credential)
-    .endpoint("<Enter Azure Load Testing Data-Plane URL>")
-    .buildClient();
-LoadTestAdministrationClient adminClient = client.getLoadTestAdministrationClient();
-TestRunClient testRunClient = client.getLoadTestRunClient();
+    .endpoint("<Enter Azure Load Testing Data-Plane URL>");
+LoadTestAdministrationClient adminClient = builder.buildLoadTestAdministrationClient();
+LoadTestRunClient testRunClient = builder.buildLoadTestRunClient();
+
+adminClient.list(null);
```

Comment:
nit: consider passing non null value for example.
here and below.

Reply:
Added filter parameters